src/app/learn/get-started/advanced/include-static-assets/index.html

```diff
@@ -48,6 +45,28 @@
     </li>
   </ol>
 
+  <stache-page-anchor>
+    Include a static asset from a global style sheet
+  </stache-page-anchor>
+
+  <ol>
+    <li>
```

Comment:
Do we need an initial step like in the previous section telling users to store the statics asset in `src/assets`? The example here is still using `/assets/img/`, so it seems like they'd still need to add the image to that folder path.